### ISSUE TITLE
process.hrtime.bigint can use v8 Fast API BigInt support

### ISSUE THREAD

**Issue by H4ad** [nodejs/performance#122]
v8 Fast APIs have support for BigInt as arguments and return values: https://github.com/v8/v8/commit/6584b6661b00e4b62ce82912b722a6dc201b7f8b

This commit hasn't landed on the node yet but once it lands, the most obvious usage can be on `process.hrtime.bigint`.

**Comment by anonrig** [comment]
@H4ad We can always backport the commit: https://github.com/nodejs/node/blob/69fb55e6b97c6c48048441d4d3ceef90ce6f2d81/doc/contributing/maintaining/maintaining-V8.md#backporting-to-active-branches

**Comment by H4ad** [comment]
I will do that, should I backport first and then create another PR for changing `hrtime` or I can do both in the same PR?

**Comment by anonrig** [comment]
> I will do that, should I backport first and then create another PR for changing `hrtime` or I can do both in the same PR?

You can do that in the same PR but make sure they are in different commits and the Pr only contains 2 commits

**Comment by H4ad** [comment]
I did try using Fast API with BigInt support but I didn't see any major improvement:

```
                                                confidence improvement accuracy (*)   (**)  (***)
process/bench-hrtime.js type='bigint' n=1000000          *      2.86 %       ±2.69% ±3.58% ±4.66%
process/bench-hrtime.js type='diff' n=1000000                  -0.33 %       ±2.25% ±3.00% ±3.92%
process/bench-hrtime.js type='raw' n=1000000                    0.46 %       ±2.01% ±2.68% ±3.48%

Be aware that when doing many comparisons the risk of a false-positive
result increases. In this case, there are 3 comparisons, you can thus
expect the following amount of false-positive results:
  0.15 false positives, when considering a   5% risk acceptance (*, **, ***),
  0.03 false positives, when considering a   1% risk acceptance (**, ***),
  0.00 false positives, when considering a 0.1% risk acceptance (***)
```

The current implementation is already well optimized and I don't think it is worth changing it for no good perf improvement.

I will close this issue but if anyone here is interested in taking a look, https://github.com/H4ad/node/tree/perf/hrtime-with-fast-api
